tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py

```diff
@@ -0,0 +1,219 @@
+    committee = spec.get_sync_committee_indices(state, spec.get_current_epoch(state))
+
+    yield 'pre', state
+
+    blocks = []
+    for _ in range(2):
+        # NOTE: need to transition twice to move beyond the degenerate case at genesis
+        block = build_empty_block_for_next_slot(spec, state)
+        # Valid sync committee signature here...
+        block.body.sync_committee_bits = [True] * len(committee)
+        block.body.sync_committee_signature = compute_aggregate_sync_committee_signature(
+            spec,
+            state,
+            block.slot - 1,
+            committee,
+        )
+
+        signed_block = state_transition_and_sign_block(spec, state, block)
+        blocks.append(signed_block)
+
+    invalid_block = build_empty_block_for_next_slot(spec, state)
+    # Invalid signature from a slot other than the previous
+    invalid_block.body.sync_committee_bits = [True] * len(committee)
+    invalid_block.body.sync_committee_signature = compute_aggregate_sync_committee_signature(
+        spec,
+        state,
+        invalid_block.slot - 2,
+        committee,
+    )
+    blocks.append(invalid_block)
+
+    expect_assertion_error(lambda: transition_unsigned_block(spec, state, invalid_block))
+
+    yield 'blocks', blocks
+    yield 'post', None
+
+
+@with_all_phases_except([PHASE0, PHASE1])
+@spec_state_test
+def test_invalid_signature_previous_committee(spec, state):
+    # NOTE: the `state` provided is at genesis and the process to select
+    # sync committees currently returns the same committee for the first and second
+    # periods at genesis.
+    # To get a distinct committee so we can generate an "old" signature, we need to advance
+    # 2 EPOCHS_PER_SYNC_COMMITTEE_PERIOD periods.
+    current_epoch = spec.get_current_epoch(state)
+    previous_committee = state.next_sync_committee
+
+    epoch_in_future_sync_commitee_period = current_epoch + 2 * spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD
+    slot_in_future_sync_committee_period = epoch_in_future_sync_commitee_period * spec.SLOTS_PER_EPOCH
+    transition_to(spec, state, slot_in_future_sync_committee_period)
+
+    pubkeys = [validator.pubkey for validator in state.validators]
+    committee = [pubkeys.index(pubkey) for pubkey in previous_committee.pubkeys]
+
+    yield 'pre', state
+
+    block = build_empty_block_for_next_slot(spec, state)
+    block.body.sync_committee_bits = [True] * len(committee)
+    block.body.sync_committee_signature = compute_aggregate_sync_committee_signature(
+        spec,
+        state,
+        block.slot - 1,
+        committee,
+    )
+
+    yield 'blocks', [block]
+    expect_assertion_error(lambda: spec.process_sync_committee(state, block.body))
+    yield 'post', None
+
+
+@with_all_phases_except([PHASE0, PHASE1])
+@spec_state_test
+def test_valid_signature_next_committee(spec, state):
+    current_epoch = spec.get_current_epoch(state)
+    next_committee = state.next_sync_committee
```

Comment:
does `next_committee == current_committee` because of genesis conditions?
To make this a more interesting test, should it also transition two periods (like the test above)

Reply:
yep, there is a "clamp" at genesis so the genesis `current` and `next` sync committees are identical.

resolved in https://github.com/ethereum/eth2.0-specs/pull/2175/commits/e518c4d04d1fb741d4b22a71d48a6f4b38582b6d